packages/website/src/data/cucumber/intro/custom-reporters.md

```diff
@@ -0,0 +1,71 @@
+## Custom Reporters
+
+ To generate an HTML view of your test reports, you can use
+[cucumber-html-reporter](https://github.com/gkushang/cucumber-html-reporter).
+
+ ### Step 1 - Installing dependencies
+
+ ```terminal
+npm install --save-dev cucumber-html-reporter npm-run-all
+```
+
+ ### Step 2 - Create a reports config file
+
+ This script is executed _after_ your test suites have completed, and sets various options on the reporter.
+
+ ```js
+// reports.conf.js
+ const reporter = require('cucumber-html-reporter');
+ const options = {
+    theme: 'simple',
+    jsonFile: 'report.json',
+    output: 'report.html',
+    reportSuiteAsScenarios: true,
+    launchReport: true,
+};
+ reporter.generate(options);
+```
+
+ For details on available options, see the
+[Cucumber HTML Reporter documentation](https://github.com/gkushang/cucumber-html-reporter#usage).
+
+ ### Step 3 - Update your npm scripts
+
+ You may have noticed in the first step we installed `npm-run-all`. This lets us run a sequence of npm scripts, while
+avoiding cross platform issues operators such as `&&` introduce. Another handy feature is the ability to continue the
+sequence when a previous script fails. This is important because we want to generate a test report even when our tests
+fail, and cause Nightwatch to throw an error.
```

Comment:
Should we mention somewhere that we can also run this as a separate process to generate the report?

Reply:
I think we can have it this way for now. Probably we can extend it at a later point of time. Thank you for the contribution @flashbackzoo .